src/index.js

```diff
@@ -0,0 +1,116 @@
+const BASE_URL = 'http://localhost:5000';
+
+const state = {};
+
+const landscapes = {
+  '80+': '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂',
+  '70-79': '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷',
+  '60-69': '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃',
+  '59-': '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲',
+};
+
+const skies = {
+  sunny: '☀️☁️☀️ ☁️ ☀️☁️ ☀️ ☁️ ☀️☁️☀️',
+  cloudy: '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️',
+  rainy: '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧',
+  snowy: '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨',
+};
+
+const increaseTemp = (event) => {
+  state.temp++;
+  refreshUI(state.temp);
+};
+
+const decreaseTemp = (event) => {
+  state.temp--;
+  refreshUI(state.temp);
+};
+
+const refreshUI = function () {
```

Comment:
Consider refactoring this method so that you have multiple methods that each only have a single responsibility. This implementation certainly works, but this method has a lot of responsibility. Sometimes it executes code unnecessarily. For example, if I change just the sky the method only _needs_ to execute lines 32 and 37. 

However in addition to changing the sky, we grab several other elements, set their values even though the UI isn't changing, and then resetting tempDisplay and landscapeDisplay. 

If refreshUI is broken up into several smaller methods that are responsible for just changing the sky or just changing the landscape/temp color then you'd have more single responsibility methods that don't run extra code when it's not needed.